geoengine/colorizer.py

```diff
@@ -238,19 +244,19 @@ class LinearGradientColorizer(Colorizer):
     under_color: Rgba
 
     @staticmethod
-    def from_response_linear(response: geoengine_openapi_client.LinearGradientWithType) -> LinearGradientColorizer:
+    def from_response_linear(response: geoengine_openapi_client.LinearGradient) -> LinearGradientColorizer:
         """Create a colorizer from a response."""
         breakpoints = [ColorBreakpoint.from_response(breakpoint) for breakpoint in response.breakpoints]
         return LinearGradientColorizer(
-            no_data_color=response.no_data_color,
+            no_data_color=to_rgba(response.no_data_color),
```

Comment:
why do you need to convert the type now? Can we adjust the type in OpenAPI so that we get a Rgba directly without conversion?

Reply:
The newer generator outputs a list with a limited length. This is different from Rgba, because that type is a tuple. Because of this a conversion is needed.

Because i reverted to the older generator, I was able to remove the type conversion.

Reply:
[9235965](https://github.com/geo-engine/geoengine-python/pull/187/commits/923596557d6bf89ccf3a42014535a797cf0ff1a4)